enterprise/query-logging/src/test/java/org/neo4j/kernel/impl/query/QueryLoggerIT.java

```diff
@@ -227,7 +228,7 @@ public void shouldLogParametersWhenNestedMap() throws Exception
         List<String> logLines = readAllLines( logFilename );
         assertEquals( 1, logLines.size() );
         assertThat( logLines.get( 0 ), endsWith( String.format(
-                " ms: %s - %s - {props: {name: 'Roland', position: 'Gunslinger', followers: [Jake, Eddie, Susannah]}}"
+                " ms: %s - %s - {props: {name: 'Roland', position: 'Gunslinger', followers: ['Jake', 'Eddie', 'Susannah']}}"
```

Comment:
This looks more correct now. I guess it could potentially affect somebody's custom query log parsing script, but that's life.